patches/@mendix+pluggable-widgets-tools+8.18.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js b/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
```

Comment:
Why do we need to patch PIW tools?

Reply:
We're using old version of the node here, so I need to apply a patch here because there's an error in the package version function on the e2e js script.